### DISCUSSION TITLE
How can I get formatting to work in a markdown/quarto file?

### DISCUSSION THREAD

**Discussion by PCesteban** [benlubas/molten-nvim#252] [Q&A]
I have everything setup, otter.nvim, quarto.nvim, I get diagnostics, but it seems like otter doesn't detect my formatter for python files. What should I do?

**Comment by benlubas** [comment]
Otter requires range formatting. Which a lot of formatters do not support. 

Also why have you come to me asking for help with two plugins I do not maintain? You'd get better help asking in the other repo..

**Reply by PCesteban** [reply]
I came up with this question following your jupyter notebook editing setup... Anyway, thanks.
